src/lib/services/transfer/service/endpoint.ts

```diff
@@ -27,8 +27,12 @@ export const get = function (endpoint_xid, options?, sdkOptions?) {
 >;
 
 /**
- * Delete an endpoint by its UUID.
- * @see https://docs.globus.org/api/transfer/endpoint/#delete_endpoint_by_id
+ * Delete a Globus Connect Personal entity by its UUID.
```

Comment:
I wonder if we should be a little more specific here that it deletes the transfer record, and not the local install.

Reply:
Going to opt for brevity here, at least for now. The resource documentation (which I've updated to https://docs.globus.org/api/transfer/gcp_management/#delete_endpoint_by_id) seems like it should be the source of truth on behavior.